src/catalog/src/durable.rs

```diff
@@ -258,13 +258,6 @@ pub trait DurableCatalogState: ReadOnlyDurableCatalogState {
         Ok(GlobalId::User(id))
     }
 
-    /// Allocates and returns a system [`ClusterId`].
```

Comment:
This was unused, yes? I quickly checked in the code and it seems so but wanted to double check.

Reply:
Yeah, it was unused.